specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/job.json

```diff
@@ -0,0 +1,931 @@
+                    "description": "The Id of the job."
+                }
+            }
+        },
+        "JobSubmissionJsonResponse": {
+            "description": "The job submission json response.",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "The Id of the created job."
+                }
+            }
+        },
+        "Profile": {
+            "description": "The object containing the job profile information.",
+            "properties": {
+                "jobFile": {
+                    "type": "string",
+                    "description": "The job configuration file."
+                },
+                "jobId": {
+                    "type": "string",
+                    "description": "The full ID of the job."
+                },
+                "jobID": {
+                    "$ref": "#/definitions/JobID",
+                    "description": "The ID of the job."
+                },
+                "jobName": {
+                    "type": "string",
+                    "description": "The user-specified job name."
+                },
+                "queueName": {
+                    "type": "string",
+                    "description": "The name of the queue to which the job is submitted."
+                },
+                "url": {
+                    "type": "string",
+                    "description": "The link to the web-ui for details of the job."
+                },
+                "user": {
+                    "type": "string",
+                    "description": "The userid of the person who submitted the job."
+                }
+            }
+        },
+        "Status": {
+            "description": "Gets or sets the object containing the job status information.",
+            "properties": {
+                "cleanupProgress": {
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,
+                    "description": "The progress made on the cleanup."
+                },
+                "failureInfo": {
+                    "type": "string",
+                    "description": "The information about any failures that have occurred."
+                },
+                "finishTime": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The time at which the job completed."
+                },
+                "historyFile": {
+                    "type": "string",
+                    "description": "The history file of the job."
+                },
+                "jobACLs": {
+                    "$ref": "#/definitions/JobACLs",
+                    "description": "The ACLs of the job."
+                },
+                "jobComplete": {
+                    "type": "boolean",
+                    "x-nullable": false,

```

Comment:
x-nullable false is not necessary. This is the default.

Reply:
`x-nullable` for basic types will be removed in the next iteration

---
In reply to: [237732282](https://github.com/Azure/azure-rest-api-specs/pull/4489#discussion_r237732282) [](ancestors = 237732282)